rpcs3/rpcs3qt/game_list_frame.cpp

```diff
@@ -448,86 +449,94 @@ void game_list_frame::Refresh(const bool fromDrive, const bool scrollAfter)
 
 		QSet<QString> serials;
 
-		for (const auto& dir : path_list) { try
+		QList<int> indices;
```

Comment:
`size_t` is much better suited for this use case I believe.

Reply:
conversions from and to size_t are a pain in the ass.
I think I implemented the most sensible solution for all of these occasions.